api/loginApiClient.module.ts

```diff
@@ -0,0 +1,27 @@
+﻿const API_URL = "https://testswitch-api-staging.herokuapp.com/candidates"; //TODO change to /signin when endpoint exists
+
+async function loginCredentialsIncorrect(): Promise<boolean>{
+    const response = await fetch(API_URL);
+    return response.status == 200;
+
+    //TODO add if(response.status == 403) then not authorized, anything else is another issue.
+}
```

Comment:
We talked about removing the body from this, changing the name to show that it tries to log in and returning a hard-coded status code for now rather than a boolean.